scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -1639,7 +1678,16 @@ function AIDriveStrategyUnloadCombine:requestToMoveOutOfWay(vehicle)
     self:onBlockingVehicle(vehicle)
 end
 
-function AIDriveStrategyUnloadCombine:moveAwayFromBlockingVehicle()
+function AIDriveStrategyUnloadCombine:requestToMoveForward(requestingVehicle)
+    self:debug('%s requests us to move forward.', CpUtil.getName(requestingVehicle))
+    local course = Course.createStraightForwardCourse(self.vehicle, 25, 0)
```

Comment:
Moving the forward distance to global constant would be nice.

Reply:
moved.